azkaban-common/src/main/java/azkaban/executor/Executor.java

```diff
@@ -28,8 +29,9 @@
   private final String host;
   private final int port;
   private boolean isActive;
-
-  // TODO: ExecutorStats to be added
+  // cached copy of the latest statistics from  the executor.
```

Comment:
renamed to serverStatistics to align with the name of the RESTful API . Please let me know if  this looks better.